src/types.mligo

```diff
@@ -140,7 +140,13 @@ type plist_direction = bool
 let prev = false
 let next = true
 
-// Proposal Doubly Linked List
+(*
+ * Proposal Doubly Linked List
+ *
+ * Behave like `OrderedSet`.
+ * When inserting a new key, it must be sure that it does not exist in the list or else
```

Comment:
> it must be sure that

"It should be ensured that" sounds a bit better..

Reply:
Updated this.